### PR TITLE
SwinModel docstring fix

### PR DESCRIPTION
**Pull request by shivanandmn**
# What does this PR do?
Docstring for `SwinModel` is added.

Fixes # (issue)

## Before submitting
- [x ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case. https://github.com/huggingface/transformers/issues/26638
- [ x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [x] Did you write any new necessary tests?

## Who can review?
@ydshieh

### CODE REVIEW

File: src/transformers/models/swin/modeling_swin.py [hunk 1]
```diff
@@ -942,6 +942,12 @@ def _set_gradient_checkpointing(self, module, value=False):
 @add_start_docstrings(
     "The bare Swin Model transformer outputting raw hidden-states without any specific head on top.",
     SWIN_START_DOCSTRING,
+    """
+        add_pooling_layer (`bool`, *optional*, defaults to `True`):
+                Whether or not to apply pooling layer(`nn.AdaptiveAvgPool1d`) after the encoder output.
```
Comment (ydshieh) [inline src/transformers/models/swin/modeling_swin.py:947]:
```suggestion
                Whether or not to apply pooling layer.
```

File: src/transformers/models/swin/modeling_swin.py [hunk 2]
```diff
@@ -942,6 +942,12 @@ def _set_gradient_checkpointing(self, module, value=False):
...
+        use_mask_token (`bool`, *optional*, defaults to `False`):
+                Whether or not create and apply mask tokens to the input embeddings.
```
Comment (ydshieh) [inline src/transformers/models/swin/modeling_swin.py:949]:
```suggestion
                Whether or not to create and apply mask tokens in the embedding layer.
```

**Review by ydshieh — COMMENTED** [review]
Ready to go once suggestion applied 🙏 

Thank you for the contribution 🚀

**Comment by shivanandmn** [comment]
@ydshieh 
I have updated changes. This is my first open-source contribution in the life.
I happy to learn from community.
Thanks

**Review by ydshieh — APPROVED** [review]
Thank you for your (first) contribution 🥳 🎉 !

**Pull request merged**